sap/5.2/modules/ROOT/pages/sap-connector-troubleshooting.adoc

```diff
@@ -304,17 +317,17 @@ Exception raised by myhost.com.ar|MULESOFT_IDOC_SEND_TEST
     at java.lang.Thread.run(Thread.java:680)
 ----
 
-==== Problem
-
-No TID handler is defined.
-
 ==== Solution
 
 Set `rfcType` to `trfc` or `qrfc` in `<sap:inbound-endpoint />`.
 
 [[error-parameter-not-supported]]
 === Parameter Not Supported
 
+==== Problem
+
+SAP extended properties must have valid names. If you provide an invalid property name you will get an error message similar to this one:
```

Comment:
```suggestion
Because SAP extended properties must have valid names, if you provide an invalid property name, you get an error message similar to this one:
```